### ISSUE TITLE
Validate accuracy and hyperparams for CPU training

### ISSUE THREAD

**Issue by ruohoruotsi** [ruohoruotsi/LSTM-Music-Genre-Classification#2]
Accuracy figures for training/dev/test set in the README were generated from a run with an older version of TF (1.2) on a GPU. 

Running the project freshly on a CPU, the following figures were obtained, which gives me pause. Investigate. Training accuracy is waay up and dev/validation is down significantly. 

What are the (extra) implications to training on the CPU vs GPU?

```Epoch 400/400
420/420 [==============================] - 3s 6ms/step - loss: 0.3547 - acc: 0.8667

Validating ...
120/120 [==============================] - 0s 4ms/step
Dev loss:   1.0411598483721416
Dev accuracy:   0.6666666865348816

Testing ...
60/60 [==============================] - 0s 1ms/step
Test loss:   1.1342438459396362
Test accuracy:   0.6000000089406967```

**Comment by NazarPonochevnyi** [comment]
I trained this model on Tensorflow 1.12.0 on Google TPU (https://en.wikipedia.org/wiki/Tensor_processing_unit). And got the same result as in README.MD. So the problem is definitely not in TF.

As for CPU, GPU, TPU: they differ only in architecture and specialization, this should not affect the result of the work, it only affects the speed of the calculations.

I think this result was due to chance.
As you know, when generating models, neuron weights are generated randomly, and are corrected in the learning process. Perhaps, your weights were corrected in the wrong direction. Try running the learning process several times.

**Comment by NazarPonochevnyi** [comment]
Also, your training results looks like a retraining of a neural network. How many epochs and batch_size have you chosen?

**Comment by ruohoruotsi** [comment]
Hi Nazar,
thank you for your comments. I was concerned that perhaps my code no longer did the correct thing and offhand couldn't localize the problem quickly. Thank you for confirming you were able to get the same accuracy, using 1.12 on the TPU.  I'll close this issue out now. 

Greetings and thanks for the pull-request!
